mlflow/utils/rest_utils.py

```diff
@@ -77,6 +79,53 @@ def http_request(
     Returns:
         requests.Response object.
     """
+    hostname = host_creds.host
+    cleaned_hostname = strip_suffix(hostname, "/")
+    url = f"{cleaned_hostname}{endpoint}"
+
+    if host_creds.auth_by_databricks_sdk:
+        from databricks.sdk import WorkspaceClient
+        from databricks.sdk.config import Config
+        from databricks.sdk.errors import DatabricksError
+
+        config = Config(retry_timeout_seconds=MLFLOW_DATABRICKS_ENDPOINT_HTTP_RETRY_TIMEOUT.get())
+        ws_client = WorkspaceClient(profile=host_creds.databricks_auth_profile, config=config)
+
+        try:
+            if method == "GET":
+                query = kwargs.get("params") or kwargs.get("json")
+                extra_kwargs = {"query": query}
+            else:
+                extra_kwargs = {"body": kwargs.get("json")}
+
+            raw_response = ws_client.api_client.do(
+                method=method,
+                path=endpoint,
+                headers=extra_headers,
+                raw=True,
+                **extra_kwargs,
+            )
+            return raw_response["contents"]._response
+        except DatabricksError as e:
```

Comment:
does DatabricksError contain request and response?

Reply:
It doesn't .